validator/src/main.rs

```diff
@@ -435,9 +435,11 @@ fn main() {
     validator_config.voting_disabled = matches.is_present("no_voting");
 
     validator_config.rpc_config.enable_fullnode_exit = matches.is_present("enable_rpc_exit");
+    validator_config.enable_ctrl_c_handler = true; 
 
     validator_config.rpc_config.drone_addr = matches.value_of("rpc_drone_addr").map(|address| {
-        solana_netutil::parse_host_port(address).expect("failed to parse drone address")
+    
```

Comment:
can you please avoid making this change

Reply:
If you take a look at line 442, I added this line "solana_netutil::parse_host_port(address).expect("failed to parse drone address")"